src/common/Backend.ts

```diff
@@ -93,17 +93,22 @@ export interface BackendError {
 export class Backend {
     readonly port: number;
 
+    abortController: AbortController;
```

Comment:
This should probably be private and readonly.

Reply:
I can't make it readonly because I make a new abortcontroller each run

Reply:
Which I'm pretty sure I have to, or it'll just always remain in an aborted state

Reply:
Yeah, you're right. It can't be readonly.